mora/service.py

```diff
@@ -413,3 +416,689 @@ def list_orgunits(orgid):
+        # 'association': create_association,
+        # 'it': create_it,
+        'contact': writing.create_contact,
+        # 'leader': create_leader,
+    }
+
+    reqs = flask.request.get_json()
+    for req in reqs:
+        role_type = req.get('type')
+        handler = handlers.get(role_type)
+        if not handler:
+            return flask.jsonify('Unknown role type'), 400
+
+        req = flask.request.json
+
+        handler(req, employee_uuid)
+
+    # TODO:
+    return flask.jsonify(employee_uuid), 200
+
+
+@blueprint.route('/e/<uuid:employee_uuid>/edit', methods=['POST'])
+def edit_employee_role(employee_uuid):
+    """Edits an employee
+
+    .. :quickref: Employee; Edit employee
+
+    :statuscode 200: The edit succeeded.
+
+    **Example Request**:
+
+    Request payload contains a list of edit objects, each differentiated
+    by the attribute 'type'. Each of these object types are detailed below:
+
+    **Engagement**:
+
+    :param employee_uuid: The UUID of the employee to be moved.
+
+    :<json string type: engagement,
+    :<json string uuid: The UUID of the engagement,
+    :<json object original: An optional object containing the original state
+        of the engagement. If supplied, the change will modify the existing
+        registration on the engagement object. Detailed below.
+    :<json object updated: An object containing the changes to be made to the
+        engagement. Detailed below.
+
+    The **original** and **updated** objects follow the same structure.
+    Every field in **original** is required, whereas **updated** only needs
+    to contain fields that need to change.
+
+    :<jsonarr string valid_from: The from date, in ISO 8601.
+    :<jsonarr string valid_to: The to date, in ISO 8601.
+    :<jsonarr string org_unit_uuid: The UUID of the associated org unit
+    :<jsonarr string job_title_uuid: The UUID of the job title of the
+        engagement
+    :<jsonarr string engagement_type_uuid: The UUID of the engagement type
+
+    .. sourcecode:: json
+
+        {
+            "type": "engagement",
+            "uuid": "de9e7513-1934-481f-b8c8-45336387e9cb",
+            "original": {
+                "valid_from": "2016-01-01T00:00:00+00:00",
+                "valid_to": "2018-01-01T00:00:00+00:00",
+                "job_title_uuid": "5b56432c-f289-4d81-a328-b878ea0a4e1b",
+                "engagement_type_uuid": "743a6448-2b0b-48cf-8a2e-bf938a6181ee",
+                "org_unit_uuid": "04f73c63-1e01-4529-af2b-dee36f7c83cb"
+            },
+            "updated": {
+                "valid_from": "2016-01-01T00:00:00+00:00",
+                "valid_to": "2019-01-01T00:00:00+00:00",
+                "job_title_uuid": "5b56432c-f289-4d81-a328-b878ea0a4e1b",
+            }
+        }
+    """
+
+    handlers = {
+        'engagement': edit_engagement
+        # 'association': edit_association,
+        # 'it': edit_it,
+        # 'contact': edit_contact,
+        # 'leader': edit_leader,
+    }
```

Comment:
Jeg tænkte på at de her skulle være URL parametre, så vi kunne have seperat dokumentation for hvert endpoint, om nødvendigt, med mindre de er 100% ens.

Reply:
Aha, det er et ønske fra frontenden…